cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataSubscriberService.java

```diff
@@ -89,6 +89,7 @@
  * Service responsible for consuming metadata messages from TMS and persist it to metadata store.
  * This is a wrapping service to host multiple {@link AbstractMessagingSubscriberService}s for lineage, usage
  * and metadata subscriptions.
+ * No transactions should be started in any of the overrided methods since they are already wrapped in a transaction.
```

Comment:
Is there a follow-up or todo to change the metadata subscribers so that they do not start their own transactions? As far as I know, they still do.

Reply:
MetadataSubcriberService already done by addressing this comments https://github.com/cdapio/cdap/pull/11104#discussion_r259158350.

Reply:
👍